lms/djangoapps/instructor/enrollment.py

```diff
@@ -28,7 +28,7 @@
 from xmodule.modulestore.exceptions import ItemNotFoundError
 
 from course_modes.models import CourseMode
-from courseware.models import StudentModule
+from lms.djangoapps.courseware.models import StudentModule
```

Comment:
@bradmerlin  This file has some duplicate imports. I know they're not your fault, but it's causing the build error now that you've touched those lines. Can you please fix?